packages/paste-core/components/separator/__tests__/index.spec.tsx

```diff
@@ -1,76 +1,119 @@
 import * as React from 'react';
-import {render} from 'react-dom';
-import * as renderer from 'react-test-renderer';
-import {render as testRender} from '@testing-library/react';
+import {matchers} from 'jest-emotion';
+import {render, screen} from '@testing-library/react';
 import {Theme} from '@twilio-paste/theme';
+import {CustomizationProvider} from '@twilio-paste/customization';
 // @ts-ignore typescript doesn't like js imports
 import axe from '../../../../../.jest/axe-helper';
 import {Separator} from '../src';
 
+expect.extend(matchers);
+
 describe('Separator', () => {
-  it('should render', () => {
-    const {getByRole} = testRender(<Separator orientation="horizontal" />);
-    expect(getByRole('separator')).not.toBeNull();
-  });
-  it('should render set horizontal orientation', () => {
-    const {getByRole} = testRender(<Separator orientation="horizontal" />);
-    expect(getByRole('separator').getAttribute('aria-orientation')).toBe('horizontal');
-  });
-  it('should render set vertical orientation', () => {
-    const {getByRole} = testRender(<Separator orientation="vertical" />);
-    expect(getByRole('separator').getAttribute('aria-orientation')).toBe('vertical');
-  });
-  it('should set horizontal margins', (): void => {
-    const tree = renderer
-      .create(
-        <Theme.Provider>
+  describe('Render', () => {
+    it('should render', () => {
+      const {getByRole} = render(<Separator orientation="horizontal" />);
+      expect(getByRole('separator')).not.toBeNull();
+    });
+
+    it('should render set horizontal orientation', () => {
```

Comment:
NIT: Maybe this could be `should set "aria-orientation" to "horizontal" when the value of orientation prop is "horizontal"`
 Or, for something shorter `should render correctly when the value of orientation prop is horizontal`. The wording feels off here.